support-frontend/build.sbt

```diff
@@ -25,6 +25,7 @@ libraryDependencies ++= Seq(
   "io.circe" %% "circe-generic" % circeVersion,
   "io.circe" %% "circe-generic-extras" % circeVersion,
   "io.circe" %% "circe-parser" % circeVersion,
+  "io.circe" %% "circe-optics" % circeVersion,
```

Comment:
this isn't going into support-workers so not too important, but I'm interested how much this increases the artifact size.  It seems pull in monocole, cats-macro based on my look at the maven metadata.

Reply:
Good question.
It's gone from 154M to 155M. I think we can live with this given it's not a lambda?

Reply:
👍